app_logs/aws/cloudwatch/provider.go

```diff
@@ -20,6 +20,7 @@ import (
 
 var (
 	logger               = log.New(os.Stderr, "", 0)
+	infoLogger           = log.New(logger.Writer(), "    ", 0)
```

Comment:
Does the space in quotes effectively indent any logs sent through this logger?

Reply:
That's correct.